### ISSUE TITLE
Prepare for migrate to SPA frontend.

### ISSUE THREAD

**Issue by VovaStelmashchuk** [hacklabkyiv/prismo#6]
For create good user experiense in an admin panel, we need to rewrite the frontend part of our product to the sigle page application. For unblock future work of creating SPA, we need to create the rest full API based on json and other protocol. All API for the admin FE need to provides by our backend service as REST full api.

**Comment by leechwort** [comment]
We can say this is done! @temhota tells that now it can be SPA!
